### PR TITLE
Adjust header layout

### PR DESCRIPTION
**Pull request by gustaveen**
## What?

Adjust header layout to be consistent between market-web, onboarding and embark

## Why?

Because it removes ui glitches in the Header :)

header

Related PRs:
https://github.com/HedvigInsurance/market-web/pull/556
https://github.com/HedvigInsurance/embark/pull/366

### CODE REVIEW

File: src/client/components/TopBar.tsx [hunk 1]
```diff
@@ -42,12 +42,15 @@ export const TopBarFiller = styled.div`
 const Container = styled.div`
   width: 100%;
   height: 100%;
-  max-width: 80rem;
-  padding: 0 2rem !important;
```
Comment (gustaveen) [inline src/client/components/TopBar.tsx:46]: @palmenhq was there a reason for adding `!important`? Couldn't see that it was overridden when removing it :)

Comment (palmenhq) [reply src/client/components/TopBar.tsx:46]: 🤷  if i remember correctly embark had some overrides but maybe they have been removed or idk. If it looks fine anyways i guess its fine anyways :D

**Review by palmenhq — APPROVED** [review]
🆒

File: src/client/components/TopBar.tsx (hunk 1 above)
Comment (gustaveen) [reply src/client/components/TopBar.tsx:46]: Cool! I tried it in embark and it looked fine, will double check once it's out in staging 🙌

**Pull request merged**